feed-keys.js

```diff
@@ -9,11 +9,11 @@ const na = require('sodium-native')
 module.exports = class FeedKeys {
   constructor (keys) {
     if (!isValid(keys.public)) {
-      throw new Error(`FeedKeys expects a 'public' key of form <base64>.ed25119, got ${keys.public}`)
+      throw new Error(`FeedKeys expects a 'public' key of form <base64>.ed25119 or <base64>.bbfeed-v1, got ${keys.public}`)
```

Comment:
This change is correct in relation to the other changes in this file. But I doubled checked our bendy butt stuff and we actually only modify the id, so I'll push up a change to tidy this up. We don't really want this if kind of code where we have to change a bunch of libraries to support a new feed format. Luckily we have the `curve` attribute that we can use to make bbfeed keys behave like ed25519 for things like ssb-keys. Because that is really what they are deep down :)

Reply:
Yeah I considered using curve to print the suffix, but i didn't know what bb would have.
Also is it same to assume base64 encoding on all feedId

But I'm with you, I dislike the "special case" code. I had filed this under "explicit and a bit painful, adding a third will tip this over into _must generalise_"

Reply:
Yes they should all be base64 encoded.